app/src/main/res/layout/quicksettings_website_info.xml

```diff
@@ -12,6 +12,18 @@
     android:layout_height="wrap_content"
     android:orientation="vertical">
 
+    <TextView
+        android:id="@+id/title"
+        style="@style/QuickSettingsText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="8dp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
```

Comment:
I think this layout got switch from a `ConstraintLayout` to a `LinearLayout` at some point. All the `app:layout_contraint...` attributes are useless in a `LinearLayout` and should be removed.